### DISCUSSION TITLE
Taro 打包编译

### DISCUSSION THREAD

**Discussion by Krstyl** [NervJS/taro#10308] [Q & A]
我想Taro打包的时只打包成 CSS/JS两个文件夹，应该怎么配置?

**Comment by luckyadam** [comment]
可以按照自己的想法修改 webpack 配置

**Reply by Krstyl** [reply]
对webpack了解不是很够，在网上没有查询到所需的情况，也没有找到设置的地方。如果大佬有空，麻烦指导一下。需求是打包的时候把CSS/JS各集合到一个文件中。
